index.js

```diff
@@ -3,16 +3,47 @@ const cards = document.querySelectorAll('.memory-card')
 const restartBtn = document.querySelector('#restart-btn')
 const timer = document.querySelector('#timer')
 let counter = 0;
+let memoryCounter = 5;
 
-//increasing the counter
-const interval = setInterval(function(){
-    counter++;
-    console.log()
-    timer.innerHTML = "<b>" + counter + "</b>";
+// Timing the counters
+const decreaseTimer = setInterval(function(){
+    if(memoryCounter>0){
+        memoryCounter--;
+        viewAllCards();
+        console.log()
+        timer.innerHTML = "<b>" + memoryCounter + "</b>";
+    }
+    else{
+        if(counter==0){
+            cards.forEach(card=>{
+                card.addEventListener('click',flipCard);
+            })
+        }
+        counter++;
+        console.log()
+        timer.innerHTML = "<b>" + counter + "</b>";
+    }
   }, 1000);
```

Comment:
add comments explaining the logic, this code can be simplified, look for possibility of using SetTimeout